test/integration/targets/include_parent_role_vars/aliases

```diff
@@ -0,0 +1 @@
+hidden
```

Comment:
Include a comment indicating what this target is used for since this isn't something we normally do.

Reply:
Comment added, thanks.